learn/tanssi/services-payment.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Services Payment
+description: Appchains deployed through Tanssi benefit from block production services provided by a set of node operators, which are compensated with Tanssi tokens.
+---
+
+# Services Payments {: #technical-features-of-tanssi }
+
+## Introduction {: #introduction }
+
+As presented in the [Overview](/learn/tanssi/overview/){target=\_blank} article, Tanssi is an infrastructure protocol that addresses the complexities and high costs associated with setting up and maintaining blockchain infrastructure, streamlining the deployment of appchains. This protocol brings benefits for both participants:
+
+- **Appchains** - teams can focus on the core logic of their product, the UX, and the UI without dealing with the challenges of infrastructure bootstrapping and its management
+- **Node operators** - bearing with the responsibility of keeping their hardware and software configuration in optimal conditions, they are rewarded with Tanssi tokens for executing transactions and producing blocks on behalf of Tanssi and Tanssi appchains
+
+The [block production as a service](/learn/tanssi/technical-features/#block-production-as-a-service){target=\_blank} mechanism randomly selects a subset of a decentralized set of nodes and assigns them to provide services to Tanssi and the Tanssi-powered appchains for a limited period of time (session). Once the session ends, the Tanssi protocol will again select the set of block producers needed to serve Tanssi and all the active Tanssi appchains and re-assign them.
+
+All this operation carries costs that must be covered by the appchains that want to obtain the services. This article covers the general aspects of those costs and associated service payments.
+
+## Costs to Cover  {: #costs-to-cover }
+
+There are three main costs associated with block production as a service that any appchain must cover using Tanssi tokens to deploy successfully and get the block production services:
+
+- **Registration deposit** - is the initial deposit that is locked from the account that signs the appchain registration transaction
+- **Block producers assignment** - every time the Tanssi protocol assigns block producers, which happens once per session, a fixed fee is charged. This discourages a faulty appchain (whose runtime logic fails to produce valid transitions and, therefore, a valid block) from requesting being assigned nodes that will not be able to produce valid blocks
+- **Block production** - appchains need to pay for each block that is produced on their behalf. Since the protocol selects and assigns the block producers on a per-session basis, appchains must have enough funds to cover all the blocks to be produced in an entire session to be served
+
+On Dancebox, the Tanssi testnet, the current configuration is the following:
```

Comment:
```suggestion
On Dancebox, the Tanssi TestNet, the current configuration is the following:
```

I suggest removing this as this is short-lived. Once Tanssi is in Mainnet we just add tabs to the table. Maybe changing it to "The current configuration is set as follows:"